docs/source/design/credentials.rst

```diff
@@ -28,11 +33,57 @@ These locations are tried in the following order, and the first successful looku
    As with any other specification of configuration items, environment variables can be used to set or override credentials.
    Variable names take the form of ``DATALAD_CREDENTIAL_<NAME>_<COMPONENT>``, and standard replacement rules into configuration variable names apply.
 
-2. DataLad uses the `keyring` package https://pypi.org/project/keyring to connect to any of its supported back-ends for setting or getting credentials.
-   This provides support for credential storage on all major platforms, but also extensibility, providing 3rd-parties to implement and use specialized solutions.
+2. DataLad uses the `keyring` package https://pypi.org/project/keyring to connect to any of its supported back-ends for setting or getting credentials,
+   via a wrapper in ``datalad.support.keyring_``. This provides support for credential storage on all major platforms, but also extensibility, providing 3rd-parties to implement and use specialized solutions.
 
 When a credential is required for operation, but could not be obtained via any of the above approaches, DataLad can prompt for credentials in interactive terminal sessions.
 Interactively entered credentials will be stored in the active credential store available via the ``keyring`` package.
+Note, however, that the keyring approach is somewhat abused by datalad. The wrapper only uses ``get_/set_password`` of ``keyring`` with the credential's ``FIELDS`` as the name to query (essentially turning the keyring into a plain key-value store) and "datalad-<CREDENTIAL-LABEL>" as the "service name".
+With this approach it's not possible to use credentials in a system's keyring that were created by other means.
```

Comment:
not entirely clear what "other means" means here.  I can go and add credential with that name using regular mechanisms.  I guess more specific would be
```suggestion
With this approach it's not possible to use credentials in a system's keyring that were defined by other, datalad unaware software.
```
or alike?

Reply:
Well, `git` wouldn't be the best example, since we now have a way to circle through git-credential even if that is ending up in a keyring again. But agree - will change wording.